test/prog/dftb+/tests

```diff
@@ -107,6 +107,8 @@ scc/HH_custom-ref                      #? MPI_PROCS <= 1
 spin/Fe4                               #? MPI_PROCS == 2 or not WITH_MPI
 spin/Fe4_commonFermi                   #? MPI_PROCS <= 2
 
+solvation/gb_ionpair                   #? MPI_PROCS <= 1
```

Comment:
Worth adding a solvent = ??? case as well ?

Reply:
With the currently implemented features this is just a shortcut for a dielectric constant. It is more testing the parser than the implementation, combined with non-default radii with non-default units it might be worth anyway.